napari/utils/validators.py

```diff
@@ -1,4 +1,4 @@
-from collections.abc import Collection, Generator
+from collections.abc import Collection, Generator, Iterable
```

Comment:
```suggestion
from collections.abc import Collection, Generator
from typing import Iterable
```

Reply:
(collections useful for isinstance checks, but prefer `typing` module for type hints)